package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "build": "webpack --progress --color --config webpack.prod.config.js",
-    "dev": "node --harmony server/index.js",
+    "dev": "nodemon server/index.js",
```

Comment:
Do we need nodemon? We have hot reloading already, without it.

Reply:
I see now. The webpack hot reloader doesn't reload with changes to the server. We can keep that. I recommend doing npm install and commit and push again so that the package-lock is updated for all users.

Reply:
@webtechalex The package-lock.json had been pushed with this PR. The first file is that one.